checkov/json/runner.py

```diff
@@ -0,0 +1,73 @@
+import json
+import logging
+import os
+
+from checkov.common.output.record import Record
+from checkov.common.output.report import Report
+from checkov.common.runners.base_runner import BaseRunner, filter_ignored_paths
+from checkov.common.parsers.json import parse
+from checkov.json.registry import registry
+from checkov.runner_filter import RunnerFilter
+
+
+class Runner(BaseRunner):
+    check_type = "json"
+
+    def run(self, root_folder=None, external_checks_dir=None, files=None, runner_filter=RunnerFilter(),
+            collect_skip_comments=True):
+        report = Report(self.check_type)
+        definitions = {}
+        definitions_raw = {}
+        parsing_errors = {}
+
+        def load_files(files_to_load, filename_fn=None, key_fn=None):
+            for file in files_to_load:
+                f = filename_fn(file) if filename_fn else file
+                key = key_fn(f) if key_fn else f
+                (definitions[key], definitions_raw[key]) = parse(f)
```

Comment:
@BrentSouza Any reason to put this as a nested function instead of a function of `Runner`?

Reply:
I felt it was only relevant in the context of the call to `run` and not so much for the Runner itself so I nested it. I can move it if you want to keep a consistent style.

Reply:
@BrentSouza Got it. Would prefer if you moved it 🙏 
Make it private `_load_files` if you feel it is relevant only inside the class